resources/definitions/GEEETECH_A20T.def.json

```diff
@@ -0,0 +1,59 @@
+{
+    "version": 2,
+    "name": "GEEETECH A20T",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Amit L",
+        "manufacturer": "GEEETECH TECH.",
+        "file_formats": "text/x-gcode",
+        "has_materials": true,
+        "machine_extruder_trains":
+        {
+            "0": "GEEETECH_A20T_1",
+            "1": "GEEETECH_A20T_2",
+            "2": "GEEETECH_A20T_3"
+        }
+
+    },
+
+    "overrides": {
+        "machine_name": { "default_value": "GEEETECH A20T" },
+        "machine_width": {
+            "default_value": 255
+        },
+        "machine_height": {
+            "default_value": 255
+        },
+        "machine_depth": {
+            "default_value": 255

```

Comment:
In the [specifications of this printer](https://www.geeetech.com/geeetech-a20t-3in1out-mix-color-3d-printer-p-1108.html) it claims that the size is only 250x250x250 for the T edition. Did you make this bigger intentionally? It could damage the printer if it goes out of bounds.